lib/evm_database.rb

```diff
@@ -58,15 +58,29 @@ def self.seedable_plugin_classes
     Vmdb::Plugins.flat_map { |p| p.try(:seedable_classes) }.compact
   end
 
+  @@seeding = false
+  def self.seeding?
+    defined?(@@seeding) && @@seeding
+  end
+
+  def self.with_seed
+    @@seeding = true
+    yield
+  ensure
+    @@seeding = false
+  end
```

Comment:
This should probably use class instance vars as per rubocop.  Or perhaps `class_attribute`